tests/modules/nf-core/ashlar/zero_uuid.nf

```diff
@@ -0,0 +1,21 @@
+nextflow.enable.dsl=2
+
+process ZERO_UUID {
+
+    input:
+    val(file_in)
```

Comment:
I was using `val` because I want the path to the output tif file from the ashlar module so that I can run the `dd` command on it to zero out the UUID.  Perhaps I should change `file_in` to `meta` because it's really taking a meta map as input.  Actually, now that I think of it, it seems like the emit should be changed from `tif` to `meta` because that's really what's being output from the ashlar module.  What do you think @adamrtalbot?

This is our first module, so please let me know if any of this should be done differently.  Thanks!

Reply:
Hmm, that gives me an error on `tif`:
```
CRITICAL Error running test workflow (exit code 1)                                                                       
         N E X T F L O W  ~  version 23.04.1                                                                             
         Launching `./tests/modules/nf-core/ashlar/main.nf` [festering_marconi] DSL2 - revision: ce57a5de86              
         ERROR ~ No such variable: tif                                                                                   
                                                                                                                         
          -- Check script 'tests/modules/nf-core/ashlar/main.nf' at line: 12 or see '.nextflow.log' file for more details
```
I've pushed the changes that I made.  Any idea what I did wrong?  Thanks!

I'm also clearly not understanding the difference between ASHLAR.out and ASHLAR.out.tif.  They both seem to be the whole meta map output from ASHLAR, but it's hard for me to tell.  I running tests with `nf-core modules create-test-yml ashlar`.  I guess I should just be running each individual test with `nextflow run`.  Is there a better way to run tests so that you can see more debugging information?

Reply:
I see your updates correcting this further down the page.  It makes much more sense now.  Thanks for explaining it!